v2/internal/plugins/collector/proxy/manager.go

```diff
@@ -392,6 +408,20 @@ func (cm *ContextManager) AllowValuesAtAnyNamespaceLevel() {
 	cm.metricsDefinition.AllowValuesAtAnyNamespaceLevel()
 }
 
+func (cm *ContextManager) SetGlobalMetricPrefix(prefix string, removePrefixFromOutput bool) error {
+	if len(prefix) == 0 {
```

Comment:
looks like the prefix has to contain "/" at the beginning so maybe 

```suggestion
	if len(prefix) < 1 {
```

Reply:
changed to len < 2. At least 1 separator, at least 1 char